runtime-k3d/src/deployment/build_vehicleapp.sh

```diff
@@ -13,8 +13,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-APP_NAME=$(echo $VELOCITAS_APP_MANIFEST | jq .Name | tr -d '"' | tr '[:upper:]' '[:lower:]')
-DOCKERFILE_FILE="$(echo $VELOCITAS_APP_MANIFEST | jq .Dockerfile | tr -d '"')"
+APP_NAME=$(echo $VELOCITAS_APP_MANIFEST | jq .name | tr -d '"' | tr '[:upper:]' '[:lower:]')
+DOCKERFILE_FILE="./app/Dockerfile"
```

Comment:
Containerization is part of K3D deployment. We can think about moving app/Dockerfile into deployment-k3d altogether and remove it from our template repository.

Reply:
I would not do that, the Container is basically our main delivery and hiding this now from the developers is not a good idea IMO. In the PS Hackathon some of the guys also tried to build the container without VS Code, that would also be impossible.

Reply:
Than again you could argue that doing things without VSCode is not endorsed by us right now. We need to be in control of as many things as possible in order to provide bugfixes and features later on. 

Thinking about Northstart, which is totally a support option for us, we won't even have a Dockerfile anymore.

But I agree this is not in scope of this PR, I am sorry my comment was not clear on this. We should have a discussion about this abstraction.

Reply:
Just adding some thoughts (maybe not fully refined):
* I don't think containerization belongs to K3D because it could be also run in another container management system or as we already do - "locally"
* Having the proposal of Dominic in mind, we could also think about "pure source code" repositories (for compiled languages as C++ or Java) because building is also already some kind of deployment ...